master/internal/core.go

```diff
@@ -580,25 +580,28 @@ func (m *Master) Run(ctx context.Context) error {
 	template.RegisterAPIHandler(m.echo, m.db, authFuncs...)
 
 	if m.config.Telemetry.Enabled && m.config.Telemetry.SegmentMasterKey != "" {
-		if telemetry, err := telemetry.NewActor(
+		if telemetryActor, tErr := telemetry.NewActor(
```

Comment:
what's the point of telemetry -> telemetryActor? ask because it's kinda a go-ism to not name things their [type](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_name_your_variables_for_their_types)

Reply:
This came up at the same time as when there started being errors about shadowing (err -> tErr). It also seemed bad practice to have a variable reuse a name used by a package. I'll switch this back and see if a specific failure comes back.

Reply:
ah, yeah, it's a package name. mostly that doesn't matter but then you can't use the package later in the same scope. 🤷 idc, you can leave it this way. i was just curious why.

Reply:
Whatever error motivated it hasn't come back - perhaps a misunderstanding